src/JsonApiConnector.js

```diff
@@ -0,0 +1,24 @@
+let mainConnector = null;
+
+/**
+ * @todo handle if we want to overwrite mainConnector
+ */
+export default class JsonApiConnector {
+  constructor(params) {
+    this.url = params.url;
+    mainConnector = mainConnector || this;
+  }
+
+  static connector() {
+    if (!mainConnector) {
+      return new JsonApiConnector();
+    }
+    return mainConnector;
+  }
+
+  buildEndpoint(path) {
+    return `${this.url}/${path}`;
+  }
+}
+
+export const getMainConnector = JsonApiConnector.connector;
```

Comment:
To delete ? Replaced by Api ?

Reply:
aaaah merde j'avais zappé ce truc là on avait parlé de connecteur a un moment mais je sais plus ce qu'on voulait y mettre dedans...